filestore/api.py

```diff
@@ -153,34 +153,21 @@ def get_data(eid, handler_registry=None):
     if handler_registry is None:
         handler_registry = {}
     with _FS_SINGLETON.handler_context(handler_registry) as fs:
-        return fs.get_datum(eid)
+        return fs.retrieve(eid)
```

Comment:
Are you un-renaming this? It was renamed from retrieve to get_datum in your refactor, I believe.